multitime.c

```diff
@@ -741,7 +739,7 @@ int main(int argc, char** argv)
         }
 
         // Find a run of cmd which has not yet been executed.
-        int runi;
+        size_t runi = 0;
```

Comment:
I don't think `size_t` is the right type here. Maybe `off_t`? Either way, it should probably be a second commit, or second PR?

Reply:
To be fair I feel like it is inappropriate to have multiple single-line PRs, so I would prefer it to be a second commit. Also, it is indexing an array, so I think it needs to be an unsigned type?

Reply:
Single-line PRs are fine!

I've had a quick look at the spec, and I agree that `size_t` is OK -- but there again, it turns out that so is `int`! So I think we don't need to change this line particularly. However, I wouldn't object to a PR which did an audit of all the signed types to see if they can better be turned into unsigned.

Reply:
Ok I will look into it, and open a PR when ready.